### PR TITLE
added /me or /em implementation to /chat

### PR DESCRIPTION
**Pull request by lewiscowper**
This would need a little more work to come up with a more generic '/command' style interface, potentially splitting the message with spaces, which I may work on if we need more commands in chat, fixes #26

### CODE REVIEW

File: public/js/chat.js [hunk 1]
```diff
@@ -87,4 +91,4 @@
   socket.on('connect', function () {
     getUserData();
   });
-})();
+})();
```
Comment (isaacs) [inline public/js/chat.js]: I love trailing `\n`s and the Unix religion agrees, but let's not be zealots about it.

Comment (TheAlphaNerd) [inline public/js/chat.js]: Mr Style... that's my name.. that name again is Mr bikeshed

**Comment by lewiscowper** [comment]
I'm not sure why it still says it has merge conflicts, in the "files changed" section, you can see there are none. It may just not have been updated for me.

**Comment by TheAlphaNerd** [comment]
it looks to me like there have been other changes within chat.js causing a conflict... specifically the bit to make sure the window doesn't scroll to the bottom on updates.

I managed to get a rebase done, although if you don't have experience with git rebase you might want to just copy paste the changes into a new branch and make a fresh PR.

If you are going to rebase I would really suggest squashing your three commits into one prior to doing it.  Not only is this a general best practice (submitting one commit per change, it will make it simpler to rebase as you will only have to deal with the conflicts once.

**Pull request closed without merging**